integrations/terraform/envs/portal/config/environment_configuration/environments.yaml

```diff
@@ -0,0 +1,51 @@
+development:
+  AWS:
```

Comment:
lowercase

Reply:
Fixed with latest commit. Platform and service names will always be exported as lowercase, also reflected this in all example files.